src/session.ts

```diff
@@ -1,25 +1,73 @@
 import { NodeDriverServiceProvider } from "@mongosh/service-provider-node-driver";
 import { ApiClient } from "./common/atlas/apiClient.js";
-import config from "./config.js";
+import defaultConfig from "./config.js";
```

Comment:
```suggestion
import { config as defaultConfig } from "./config.js";
```
more subjective and I realize this was already the case with config but we should try to avoid default exports and particularly renaming default exports; makes it hard to refactor/debug later

Reply:
removed w/ the merge